src/store/initial-state.js

```diff
@@ -27,11 +27,16 @@ export const getInitialPipelineState = () => ({
     hovered: null
   },
   nodeType: {
-    ids: ['task', 'data', 'parameters'],
+    ids: ['tag', 'task', 'data', 'parameters'],
```

Comment:
⚠️⚠️⚠️  'tag' isn't a node-type. I think this might lead to some undesirable outcomes. Is this only used in the sidebar? If so, maybe it should be renamed, but I suspect it might be used elsewhere, in which case I'm amazed that it hasn't caused issues.

Reply:
Yeah I thought that could be the case which is why I was asking about this before, though no problems showed up strangely. Anyway I was planning to move it to the config instead, which has now been completed.